apps/speed.c

```diff
@@ -3639,7 +3643,7 @@ static void multiblock_speed(const EVP_CIPHER *evp_cipher, int lengths_single,
                 aad[12] = (unsigned char)(len);
                 pad = EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_AEAD_TLS1_AAD,
                                           EVP_AEAD_TLS1_AAD_LEN, aad);
-                EVP_Cipher(ctx, out, inp, len + pad);
+                dummy *= EVP_Cipher(ctx, out, inp, len + pad);
```

Comment:
not a fan of `dummy` in speed.c

Reply:
Me either. But I'm not sure that honest checking is an option.

Reply:
dishonest checking will cause compiler warnings as soon as it's updated to detect that the error value is actually ignored...

Reply:
Strange effect: it complains when I `and` the values but doesn't when I multiply them

Reply:
Rolled back speed.c, seeing what happens.